src/ProjectOrigin.WalletSystem.Server/Models/CertificatesFilter.cs

```diff
@@ -1,9 +1,11 @@
 using System;
+using ProjectOrigin.WalletSystem.Server.Services.REST.v1;
 
 namespace ProjectOrigin.WalletSystem.Server.Models;
 
-public record CertificatesFilter
+public record CertificatesFilter(SliceState State)
 {
     public DateTimeOffset? Start { get; init; }
```

Comment:
That can be accomplished the following way.
```suggestion
    public required SliceState State { get; init; }
    public DateTimeOffset? Start { get; init; }
```